e2e/testcases/csr_auth_test.go

```diff
@@ -70,6 +71,12 @@ func TestGCENode(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.SyncSource, ntopts.Unstructured,
 		ntopts.RequireGKE(t), ntopts.GCENodeTest)
 
+	if workloadPool, err := getWorkloadPool(nt); err != nil {
+		nt.T.Fatal(err)
+	} else if workloadPool != "" {
+		nt.T.Fatal(err)
```

Comment:
err is nil here. use a message

Reply:
or make getWorkloadPool error if empty?

Reply:
I'll hard code the message, `getWorkloadPool` is designed to be usable both for verifying existence/nonexistence of workload pool

Reply:
done